spec.bs

```diff
@@ -4003,6 +4077,10 @@ The <dfn for="interest group">estimated size</dfn> of an [=interest group=] |ig|
   [=interest group/priority signals overrides=]:
   1. The [=string/length=] of |key|.
   1. 8, which is the size of |value|.
+1. If |ig|'s [=interest group/seller capabilities=] is not null, then [=map/for each=]
+  |seller| → |capabilities| of |ig|'s [=interest group/seller capabilities=]:
+  1. The [=string/length=] of the [=serialization of an origin|serialization=] of |seller|.
+  1. 4, which is |capabilities|'s underlying number of enum bytes.

```

Comment:
Maybe add a note that this could change if more than 32 capabilities are added?

Reply:
Added to the definition of [=seller capabilities=]